recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: octave_kernel-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/o/octave_kernel/octave_kernel-{{ version }}.tar.gz
-  sha256: cd65d9a888991634ed43f01201ab068bec2b9cea4ae8642d353f6264f5ed6dd3
+  sha256: f9caec892d762f22aa8cac218e09e23b69c4a28a6c78cd6de95c7e80c9dfd96d
```

Comment:
We can try `noarch` with this recipe and get rid of the Travis-CI delays. Want to give it a shot?

Reply:
Sounds good